CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 ## [Unreleased]
 
+# 1.12.1
+### Added
+- `--output` flag indicates which package type you want to create deb\rpm\any fpm supported output.
+    Notice that the default is DEBIAN if you dont give the flag. 
+
+### Notice
+
+* Default is DEBIAN if you dont give the `--output` flag.
+* Now if you want to create both debian and rpm you need to call debify twice each time with different output.
+
+### Fixed
+
+* Same Configuration to deb and rpm bug that happens in Conjur Evoke CICD. 
+
+  https://app.zenhub.com/workspaces/palmtree-5d99d900491c060001c85cba/issues/conjurinc/dap-package/43
```

Comment:
i don't think we need these.

Reply:
also - i would create the issue in `debify` saying that we need this flag so that users can send different FPM params for different output types. it is better than having the issue in `dap-package`.

Reply:
Created Issue and linked